[docs] Add warnings to the VMware Cloud Director environment documentation about Edge requirements (#14994)
deckhouse/deckhouse: ee/candi/cloud-providers/vcd/docs/ENVIRONMENT.md

The Organization, VirtualDataCenter, StoragePolicy, SizingPolicy, EdgeRouter, and Catalog resources must be provided by your VMware Cloud Director service provider.

{% alert level="warning" %}

> Each VDC (Virtual Data Center) must have an Edge Gateway configured, otherwise the cluster will not be bootstrapped.
> 
> Also note that if two networks are distributed from the same Edge Gateway, the cluster bootstrap may also fail.

Reviewer: ```suggestion
Each VDC (Virtual Data Center) must have an Edge Gateway configured, and the cluster network must be connected to it.
```
